client/src/pages/FirstCycle/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Typography, useMediaQuery } from "@mui/material";
+import AddCircleOutlineOutlinedIcon from '@mui/icons-material/AddCircleOutlineOutlined';
```

Comment:
I would recommend using FAB component so it is an actual button and not an icon with a link